.devcontainer/Dockerfile

```diff
@@ -1,20 +1,21 @@
-ARG SWIFT_VERSION=5.9
+ARG SWIFT_VERSION=6.1
 # Other ARG declarations must follow FROM
 FROM swift:${SWIFT_VERSION}
 
 ARG HYLO_LLVM_BUILD_TYPE=MinSizeRel
 ARG HYLO_LLVM_BUILD_RELEASE=20240303-215025
 ARG HYLO_LLVM_VERSION=17.0.6
+ARG CMAKE_VERSION=3.29.1
 
 ENV HYLO_LLVM_DOWNLOAD_URL="https://github.com/hylo-lang/llvm-build/releases/download"
 
-RUN apt install -y gnupg
-RUN apt update
-RUN apt install -y curl libzstd-dev libzstd1 lsb-release make ninja-build tar wget zstd software-properties-common python3-pip
-
-# Get a recent cmake (https://www.kitware.com//cmake-python-wheels/)
-RUN if $(/usr/bin/which cmake) ; then apt purge --auto-remove cmake ; fi
-RUN pip3 install --upgrade cmake
+# Install the dependencies for the LLVM build.
+# Note that we do cleanup in the same layer to save disk space.
+RUN apt-get update && \
```

Comment:
Why use `&&` instead of separate `RUN` commands?

Reply:
Each RUN command creates its own layer, each of which consumes a bit of disk space. Doing an `rm` in a separate pass doesn't save any disk space but rather increases the size. (Maybe the final image is smaller, but at least the caches get bigger, I'm not sure about the internals of docker)